src/test/java/io/confluent/connect/jdbc/source/JdbcSourceTaskUpdateTest.java

```diff
@@ -366,11 +475,38 @@ public void testTimestampRestoreOffset() throws Exception {
     PowerMock.verifyAll();
   }
 
+  @Test
+  public void testTimestampAndIncrementingRestoreNoVersionOffset() throws Exception {
+    TimestampIncrementingOffset offset = new TimestampIncrementingOffset(new Timestamp(10L), 3L);
+    testTimestampAndIncrementingRestoreOffset(
+        Collections.singletonMap(SINGLE_TABLE_PARTITION, offset.toMap())
+    );
+  }
+
+  @Test
+  public void testTimestampAndIncrementingRestoreVersionOneOffset() throws Exception {
+    TimestampIncrementingOffset offset = new TimestampIncrementingOffset(new Timestamp(10L), 3L);
+    testTimestampAndIncrementingRestoreOffset(
+        Collections.singletonMap(SINGLE_TABLE_PARTITION_WITH_VERSION, offset.toMap())
+    );
+  }
+
   @Test
   public void testTimestampAndIncrementingRestoreOffset() throws Exception {
```

Comment:
This is testing when we have offsets with both protocols. Is that worth having the method include that information? E.g., `testTimestampAndIncrementingRestoreOffsetWithMultipleProtocol()`